ehrapy/plot/supervised/_feature_importances.py

```diff
@@ -0,0 +1,30 @@
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+from anndata import AnnData
+
+
+def feature_importances(adata: AnnData, key: str = "feature_importances", n_features: int = 10):
+    """
+    Plot features with greates absolute importances as a barplot.
+
+    Args:
+        adata: :class:`~anndata.AnnData` object storing the data. A key in adata.var should contain the feature
+            importances, calculated beforehand.
+        key: The key in adata.var to use for feature importances. Defaults to 'feature_importances'.
+        n_features: The number of features to plot. Defaults to 10.
+
+    Returns:
+        None
+    """
+    if key not in adata.var.keys():
+        raise ValueError(f"Key {key} not found in adata.var.")
```

Comment:
This would raise a `KeyError` anyways. Catching this is more helpful if you specify how to resolve -> calculate the feature importances first by doing blabla